about.md

```diff
@@ -0,0 +1,59 @@
+---
+layout: page
+title: About
+#external-link: //pelux.io/software-factory
+permalink: /about/
+---
+
```

Comment:
A general comment on this page is that I would prefer if it had line breaks (at char 80 or char 100). It's just a preference, not a requirement, in my opinion. Single line breaks don't break the paragraph.

Reply:
Good catch, I will fix this. I forgot to do this on this page.